frontend/src/scenes/data-warehouse/editor/sidebar/EditorSidebar.tsx

```diff
@@ -87,6 +92,19 @@ export const EditorSidebar = ({
         },
     ]
 
+    if (featureFlags[FEATURE_FLAGS.AI_HOGQL_QUERY_EDITOR]) {
+        tabs.push({
+            key: EditorSidebarTab.QueryAI,
+            label: (
+                <Tooltip title="AI">
+                    <div className="flex justify-center px-2">
+                        <IconMagicWand className="text-xl" />
+                    </div>
+                </Tooltip>
+            ),
+        })
+    }
```

Comment:
How do we handle frontend features that use stuff from `ee`? Do we simply not show it if the project is being used from the `foss` repo?